kmod/src/data.c

```diff
@@ -1661,19 +1662,24 @@ int scoutfs_data_fiemap(struct inode *inode, struct fiemap_extent_info *fieinfo,
 
 	iblock = start >> SCOUTFS_BLOCK_SM_SHIFT;
 	last = (start + len - 1) >> SCOUTFS_BLOCK_SM_SHIFT;
+	max = ext.start + ext.len;
```

Comment:
ext is uninitialized here.   This never did anything.   (and fwiw I try to avoid min/max for var names because of the min() and max() macros).  We can just remove the max business.  We have plenty of other loops that rely on ext start + len never wrapping.  ('cause they're block granular u64 and are derived from u64 bytes, they should never be that enormous).